### ISSUE TITLE
How about use the `uninstall` hook to do some cleanup tasks?

### ISSUE THREAD

**Issue by poppinlp** [jdneo/vscode-leetcode#148]
There could be some cleanup tasks such as remove the `~/.leetcode`.
Maybe we could do it through the `uninstall` hook (after a user confirm?).

**Comment by jdneo** [comment]
@poppinlp Could you explain why you want to remove `~./leetcode`?

**Comment by poppinlp** [comment]
Since i don't want something extra still left on my disk which is generated by the extension after i uninstall the extension.
I think it is a normal scenario for a program to clean up the extra data which is created by it when it is uninstalled.
But in some situations, maybe user wanna keep it. So we may add a confirm. :)

**Comment by jdneo** [comment]
@poppinlp Good idea but I think it's not a high priority one. Better to wait for more feedback.

---
If any user wants to add this feature, please click the 👍 button to highlight this issue.

**Comment by jdneo** [comment]
upstream issue: https://github.com/microsoft/vscode/issues/45474
